src/test/api_compatibility.ts

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
This is a comment about span_context.ts (which I can't comment on directly).
* `default class` was replaced in other files
* can it be just an interface?